monitoring/ceph-mixin/dashboards/ceph-nvmeof.libsonnet

```diff
@@ -119,191 +119,164 @@ local g = import 'grafonnet/grafana.libsonnet';
-          {
-            id: 'color',
-            value: {
-              fixedColor: 'green',
-              mode: 'fixed',
+          ],
+        },
+        {
+          matcher: { id: 'byName', options: 'Available' },
+          properties: [
+            {
+              id: 'color',
+              value: { 
+                fixedColor: 'green', 
+                mode: 'fixed' 
+              },
             },
-          },
-        ],
-      },
-    ]),
+          ],
+        },
+      ]),
 
-    $.timeSeriesPanel(
-      title='Ceph Health NVMeoF WARNING',
-      description='Ceph healthchecks NVMeoF WARNINGs',
-      gridPosition={ x: 8, y: 1, w: 7, h: 8 },
-      lineInterpolation='linear',
-      lineWidth=1,
-      drawStyle='line',
-      axisPlacement='auto',
-      datasource='$datasource',
-      fillOpacity=5,
-      pointSize=5,
-      showPoints='auto',
-      unit='none',
-      displayMode='list',
-      showLegend=true,
-      placement='bottom',
-      tooltip={ mode: 'multi', sort: 'desc' },
-      stackingMode='normal',
-      spanNulls=false,
-      decimals=0,
-      thresholdsMode='absolute',
-      noValue='0',
-    ).addThresholds([
-      { color: 'green', value: null },
-    ])
-    .addTarget(
-      $.addTargetSchema(
-        expr="sum(ceph_health_detail{name='NVMEOF_GATEWAY_DOWN'})",
-        format='',
-        instant=false,
-        legendFormat='NVMEOF_GATEWAY_DOWN',
-        range=true,
+      $.timeSeriesPanel( 
+        title='Gateway WARNING Health States',
+        description='Gateways in error states',
+        gridPosition={ x: 8, y: 1, w: 7, h: 8 },
+        lineInterpolation='linear',
+        lineWidth=1,
+        drawStyle='line',
+        axisPlacement='auto',
         datasource='$datasource',
+        fillOpacity=5,
+        pointSize=5,
+        showPoints='auto',
+        unit='none',
+        displayMode='list',
+        showLegend=false,
+        placement='bottom',
+        tooltip={'hideZeros': true, 'mode': 'multi', 'sort': 'desc'},
+        stackingMode='normal',
+        spanNulls=false,
+        decimals=0,
+        thresholdsMode='absolute',
+        noValue=0,
+      ).addThresholds([
+        { color: 'green', value: null },
+      ])
+      .addTarget(
+        $.addTargetSchema(
+          expr='group by (daemon_name) (ceph_cephadm_daemon_status{service_type="nvmeof"} == 0)',
+          format='',
+          instant=false,
+          legendFormat="stopped - {{ daemon_name }} ",
+          range=true,
+          datasource='$datasource',
+        )
       )
-    )
-    .addTarget(
-      $.addTargetSchema(
-        expr="sum(ceph_health_detail{name='NVMEOF_GATEWAY_DELETING'})",
-        format='',
-        instant=false,
-        legendFormat='NVMEOF_GATEWAY_DELETING',
-        range=true,
-        datasource='$datasource',
+      .addTarget(
+        $.addTargetSchema(
+          expr='group by (daemon_name) (ceph_cephadm_daemon_status{service_type="nvmeof"} == -1)',
```

Comment:
I have not documented -1, -2 in my PR, are you sure, this is what they are reporting?